porter/v1/cluster_control_plane.proto

```diff
@@ -266,6 +266,17 @@ message DeleteAppDeploymentRequest {
 // DeleteAppDeploymentResponse is the response object when removing a porter app from a given deployment target
 message DeleteAppDeploymentResponse {}
 
+// DeleteDeploymentTargetRequest is the request object when removing a deployment target from a given cluster
+message DeleteDeploymentTargetRequest {
+  int64 project_id = 1;
+  int64 cluster_id = 2;
```

Comment:
why do we need the cluster id?

Reply:
used to check if it matched the cluster on the deployment target, but this can be done elsewhere so removing for now